canu/test/aruba/test_suite.yaml

```diff
@@ -239,6 +257,10 @@
     - leaf-bmc
     - cdu
     - spine
+  csm:
```

Comment:
1.6?

Reply:
removed dhcp, dns ping tests.  DHCP test no longer worked because of ACLs.  DNS ping test does not seem useful IMO.